packages/components/src/FormField/FormField.css

```diff
@@ -233,6 +233,8 @@
 }
 
 .textarea.safari .input {
+  margin-top: var(--field--padding-top);
```

Comment:
any reason you went with margin?

it's doing that thing again where as you scroll the text feels like it disappears into the white block of margin rather than going out of view of the textarea

if we use padding, that should fix it

Reply:
I guess we might have been looking at different stories/variations, since I didn't see that originally. But I changed back to using padding instead of margin. I also, changed the label width to allow some space (minimal amount) for the scrollbar, so it shouldn't be getting cut off/hidden behind the label.